### PR TITLE
Add support of Patroni standby-cluster feature

### PR DESCRIPTION
**Pull request by CyberDem0n**
and make it possible to configure a backup location by specifying `WALE_*_PREFIX` environment variable instead of generating the path.

### CODE REVIEW

File: ENVIRONMENT.rst [hunk 1]
```diff
@@ -40,9 +42,10 @@ Environment Configuration Settings
 - **SWIFT_PASSWORD**:
 - **SWIFT_AUTH_VERSION**:
 - **SWIFT_ENDPOINT_TYPE**:
+- **WALE_SWIFT_PREFIX**: (optional) the full path to the backup location on the Swift Storage in the format swift://bucket-name/very/long/path. If not specified Spilo will generate it from WAL_SWIFT_BUCKET.
```
Comment (avaczi) [inline ENVIRONMENT.rst]: Isn't `WAL_SWIFT_BUCKET` missing here?

Comment (CyberDem0n) [reply ENVIRONMENT.rst]: Nope, it is [here](https://github.com/zalando/spilo/pull/315/files#diff-787073732367780d2de9c4a21cfbec09L36).

Comment (avaczi) [reply ENVIRONMENT.rst]: Oh.  

OTOH I think the 'Ditto' parts here and there are referring to a previous version, and some other variables came in between, see the link you gave

**Comment by avaczi** [comment]
:+1:

**Comment by avaczi** [comment]
:+1:

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**